nix/tests/expected/extensions_sql_interface.out

```diff
@@ -5888,42 +5919,6 @@ order by
  supabase_vault               | vault                    | secrets                                          | nonce
  supabase_vault               | vault                    | secrets                                          | secret
  supabase_vault               | vault                    | secrets                                          | updated_at
- timescaledb                  | _timescaledb_config      | bgw_job                                          | application_name
```

Comment:
this diff is because timescale uses 3 schemas but we weren't ordering the tables by namespace. After this PR this should not happen again